tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -1093,6 +1125,55 @@ public function test_get_data_for_save__throws_on_settings_validation_error() {
 		$widget->get_data_for_save();
 	}
 
+	public function test_get_data_for_save__throws_on_styles_stroke_prop_validation_error() {
+		// Arrange.
+		$widget = $this->make_mock_widget( [
+			'props_schema' => [
+				'string_prop' => String_Prop_Type::make()->default( '' ),
+			],
+			'settings' => [
+				'string_prop' => 'valid-string',
+			],
+			'styles' => [
+				's-1234' => [
+					'id' => 's-1234',
+					'type' => 'class',
+					'variants' => [
+						[
+							'props' => [
+								'-webkit-text-stroke' => [
+									'$$type' => 'stroke',
+									'value' => [
+										'color' => null,
+										'width' => [
+											'$$type' => 'size',
+											'value' => [
+												'unit' => 'px',
+												'size' => 'test',
+											],
+										],
+									],
+								],
+							],
+							'meta' => [
+								'breakpoint' => 'desktop',
+								'state' => null,
+							],
+						],
+					],
+				]
+			]
+		] );
+
+		// Expect.
+		$this->expectException( \Exception::class );
+		$this->expectExceptionMessage( 'Styles validation failed. Invalid keys: -webkit-text-stroke' );
+
+		// Act.
+		$widget->get_data_for_save();
+	}
+
+
```

Comment:
```suggestion
	}

```


remove one empty line